packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -5,16 +5,48 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import * as fs from 'fs';
+import * as path from 'path';
 import * as vscode from 'vscode';
+import { LanguageClient } from 'vscode-languageclient';
 import { APEX_LANGUAGE_SERVER_CHANNEL } from './channel';
 import * as languageServer from './language-server';
 
+let apexClient: LanguageClient | undefined;
```

Comment:
@DatGuyJonathan - This was what I was thinking yesterday, that we can get a handle to the debugger client/server to manually send commands over to it. I know that it can be done for the Language Client/Server.